docs/release_notes/v1.0.0-rc.3.md

```diff
@@ -0,0 +1,512 @@
+  
+# Dapr 1.0.0-rc.3
+
+We're happy to announce the release of Dapr 1.0.0-rc.3!
+
+We would like to extend our [thanks to all](#Acknowledgements) new and existing contributors who helped make this release happen.
+
+This is a release candidate version as we progress towards Dapr v1.0.0. We appreciate you trying Dapr and providing feedback on this release.
+
+See [this](#upgrading-to-dapr-100-rc3) section on upgrading Dapr to version 1.0.0-rc.3.
+
+## Highlights ##
+
+* Pub/Sub [per message time-to-live](https://v1-rc3.docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-message-ttl/)
+Dapr enables per-message time-to-live (TTL) for all pub/sub components. Applications can set time-to-live per message, and subscribers will not receive those messages after expiration.
+* Sidecar to app authentication
+* [MySQL output binding](https://v1-rc3.docs.dapr.io/operations/components/setup-bindings/supported-bindings/mysql/)
+* [MySQL state store](https://v1-rc3.docs.dapr.io/operations/components/setup-state-store/supported-state-stores/setup-mysql/)
+* Component metadata's random value via `{uuid}` marker
+* [Upgrade CLI command to upgrade the Dapr control plane](https://github.com/dapr/cli#upgrade-dapr-on-kubernetes)
+* Numerous improvements to the [Dapr Dashboard](https://github.com/dapr/dashboard) 
+* Added [dapr-distributed-calendar sample]9https://github.com/dapr/samples/tree/master/dapr-distributed-calendar)
```

Comment:
Typo

```suggestion
* Added [dapr-distributed-calendar sample](https://github.com/dapr/samples/tree/master/dapr-distributed-calendar)
```

Reply:
The source of this file is hosted someplace else and synced here. I fixed the source based on your suggestion. Thanks!